app/js/components/LabResultEntry.jsx

```diff
@@ -141,6 +141,8 @@ export class LabResultEntry extends PureComponent {
                 widget="dropdown"
                 concept={CONSTANTS.labResultsDidNotPerformReasonQuestion}
                 path="did-not-perform-dropdown"
+                defaultValue=" "
```

Comment:
I added this so the dropdown would have an empty initial display name 
<img width="438" alt="screenshot 2018-10-16 at 2 26 50 pm" src="https://user-images.githubusercontent.com/26272732/47019695-978caa80-d14f-11e8-91b5-bc9657300fc1.png">